### ISSUE TITLE
exceeds max pixel size of 0, you can change the value in config pimcore.assets.image.max_pixels

### ISSUE THREAD

**Issue by ascheider** [BlackbitDigitalCommerce/pimcore-data-director#130]
Pimcore 11.1.1
Data Director 3.5

If i import the asset and use the template: Fetch Image from file / URL
![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/6016485/7d235274-6340-4ff5-b032-c4dc7ba5658e)

i get the error, that max_pixels is exceeded.
Pimcore converts it to the integer, but data director set it ti the INF
![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/6016485/31093b4d-986b-4ea5-8665-72babed47f81)

![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/6016485/74823a46-7fa1-40c3-bdc4-d02fb2df696a)

Basically int of INF is 0 and no upload from url is possible currently.

**Comment by BlackbitDevs** [comment]
Is fixed in 3.5.15 - now `max_pixels` gets set to `PHP_INT_MAX` instead if `INF`.
